scripts/deployment/rinkeby_testnets.json

```diff
@@ -9,14 +9,16 @@
             "name": "boba testnet",
             "rpc": "https://rinkeby.boba.network/",
             "chain_id": 28,
-            "messenger_contract_address": "0xF10EEfC14eB5b7885Ea9F7A631a21c7a82cf5D76",
+            "proof_submitter": "OptimismProofSubmitter",
+            "l2_to_l1_messenger": "0xF10EEfC14eB5b7885Ea9F7A631a21c7a82cf5D76",
             "finalization_time": 604800
         },
         {
             "name": "metis testnet",
             "rpc": "https://stardust.metis.io/?owner=588",
             "chain_id": 588,
-            "messenger_contract_address": "0xBa6f1aAF3F909F208b224F3985eCda745FA9101A",
+            "proof_submitter": "OptimismProofSubmitter",
+            "l2_to_l1_messenger": "0xBa6f1aAF3F909F208b224F3985eCda745FA9101A",
```

Comment:
here is some inconsistent naming. Anywhere else the term `l2_messenger` or `l1_messenger` is used. I think consistency would be nicer.

Reply:
Good catch! That's a leftover from a previous iteration of the patches and is clearly wrong.

Reply:
Fixed.